src/shared/components/query/QueryStore.ts

```diff
@@ -807,24 +777,52 @@ export class QueryStore
 
 	@computed get unknownStudyIds()
 	{
+		const selectableStudies = this.selectableIdToPhysicalIdsMap.result;
 		let ids:string[] = this._selectedStudyIds.keys();
-		const selectableStudies = this.selectableStudiesSet;
-		ids = ids.filter(id=>!(id in selectableStudies));
-		return ids;
+		return ids.filter(id=>!(id in selectableStudies));
 	}
 
 	@computed get selectedStudies_totalSampleCount()
 	{
-		return this.selectedStudies.reduce((sum:number, study:CancerStudy) => sum + study.allSampleCount, 0);
+		const result:{[id:string]:number} = {};
+		const virtualStudySamples:{[id:string]:string[]} = {};
+
+		this.selectedStudies.forEach(study => {
+			//merge samples for a study across all virtual studies 
+			if(this.isVirtualCohort(study.studyId)){
+				this.virtualStudiesSet[study.studyId] 
+				&& this.virtualStudiesSet[study.studyId]
+						.data
+						.studies
+						.forEach(study => {
+							let samples = virtualStudySamples[study.id] || []
+							//samples may contain duplicates
+							virtualStudySamples[study.id] = samples.concat(study.samples)
+						})
+			}else{
+				//add selected physical studies samples count to result
+				result[study.studyId] =study.allSampleCount;
+			}
+		})
+		//if the physical study in virtual study is not present result object, then 
+		//add the study and samples count into result object
+		Object.keys(virtualStudySamples).forEach(id => {
+			if(!result[id])
+				result[id] = _.uniq(virtualStudySamples[id]).length
+		});
+		return Object.keys(result).reduce((sum, id) => sum + result[id], 0);
 	}
 
 	public isVirtualCohort(studyId:string):boolean {
 		// if the study id doesn't correspond to one in this.cancerStudies, then its a virtual cohort
 		return !this.cancerStudyIdsSet.result[studyId];
 	}
 
-	public isTemporaryVirtualCohort(studyId:string):boolean {
-		return this.temporaryVirtualCohortId.isComplete && this.temporaryVirtualCohortId.result === studyId;
+	public isDeletedVirtualStudy(studyId:string):boolean {
+		if(this.isVirtualCohort(studyId) && this.deletedVirtualStudies.indexOf(studyId) > -1){

```

Comment:
how come we need to know that something is a deleted virtual study?  seems like this should be handled simple by removal of study from collection of studies.